### ISSUE TITLE
Mass edit agents

### ISSUE THREAD

**Issue by manc74** [amidaware/tacticalrmm#1149]
It would be great to be able to select multiple agents to edit (e.g. assign to a different site or client).

**Comment by wh1te909** [comment]
yes definitely need to add this. was going to have a mass action screen similar to this https://github.com/amidaware/tacticalrmm/issues/976 where you can perform bulk actions on agents. atm the only way to do it is from management commands or thru the api

**Comment by StegaNetworks** [comment]
+1 for this, lack of ability to select multiple agents is a big hindrance for day-to-day use.

Would love to be able to select a bunch of agents, and right click to edit / move / run script / delete / etc…

**Comment by pmehlb** [comment]
+1 for this as well, being able to select (and act on) multiple agents is the only pain I've run into with TRMM

**Comment by PeetMcK** [comment]
+1 ... with the time out for offline agents this is a truly unbearable process.  Are there any instructions to do it locally on the server though the terminal?

**Comment by zoltan151** [comment]
+1

**Comment by crispyunicornsteak** [comment]
**1+**
edit, move, run script, windows updates; that would be a HUGE feature!

**Comment by silversword411** [comment]
Multi select is in progress. We'll be making functions up as we can. Some need backend new functions
